src/hooks/historikk/useHistorikk.ts

```diff
@@ -26,6 +27,8 @@ export function useHistorikk(): HistorikkHook {
     isError: isMotebehovError,
   } = useHistorikkMotebehovQuery();
 
+  const motebehovNyHistorikk = useMotebehovHistorikk();
```

Comment:
Hvorfor "Ny"?

Reply:
Driver og tester litt i dev og trenger begge to midlertidig for å kryssjekke litt - målet er å fjerne bruken av den dataen som kommer fra queryen på L28 her og ikke ha ny i navnet ☺️